deploy/helm/sumologic/conf/logs/logs.source.containers.conf

```diff
@@ -8,53 +8,37 @@
     time   ${[record["log"].split(/[\n\t]+/)[0]].map! {|item| JSON.parse(item)["time"]}.join("")}
   </record>
 </filter>
+# match all  container logs and label them @NORMAL
 <match containers.**>
   @type relabel
   @label @NORMAL
 </match>
 <label @NORMAL>
+  #  only match fluentd logs based on fluentd container name
+  {{ printf "<filter **%s**>" (include "sumologic.fullname" .) }}
+    # only ingest fluentd logs of levels: {error, fatal} and warning messages if buffer is full
+    @type grep
+    <regexp>
+    key log
+    pattern /\[info\]|\[fatal\]|drop_oldest_chunk/
```

Comment:
Wait, we want to ingest info logs here?

Reply:
good catch, I had changed that to test locally. Will fix it to be `error`.